runtime/commonMain/src/kotlinx/serialization/json/JsonConfiguration.kt

```diff
@@ -88,33 +82,25 @@ public data class JsonConfiguration @UnstableDefault constructor(
          * we can't guarantee backwards compatibility and it is likely to change.
          */
         @JvmStatic
-        @UnstableDefault
-        public val Default: JsonConfiguration = JsonConfiguration()
+        public val Default: SubtypeToDetectDefaults = SubtypeToDetectDefaults()
 
         /**
          * Stable [JsonConfiguration] that is guaranteed to preserve its semantics between releases.
          * To have a stable base in your [Json] configuration you can use `Stable.copy(param = ...)`
          */
-        @OptIn(UnstableDefault::class)
         @JvmStatic
-        public val Stable: JsonConfiguration = JsonConfiguration(
-            encodeDefaults = true,
-            ignoreUnknownKeys = false,
-            isLenient = false,
-            serializeSpecialFloatingPointValues = false,
-            allowStructuredMapKeys = true,
-            prettyPrint = false,
-            unquotedPrint = false,
-            indent = defaultIndent,
-            useArrayPolymorphism = false,
-            classDiscriminator = defaultDiscriminator
-        )
+        public val Stable: SubtypeToDetectDefaults = SubtypeToDetectDefaults()
     }
 }
 
-@Deprecated(level = DeprecationLevel.ERROR, message = "This named parameters were deprecated." +
-        "Intstead of 'unquoted' please use 'unquotedPrint'. 'strictMode' was splitted into" +
-        "'ignoreUnknownKeys', 'isLenient' and 'serializeSpecialFloatingPointValues'")
+@Suppress("DEPRECATION_ERROR")
+public class SubtypeToDetectDefaults : JsonConfiguration()
```

Comment:
No on both. Internals will effectively prevent migration and deprecation adds additional confusion and makes replacement even harder (I've tried it first).
These types are useless anyway and will be removed in 1.0.0

Reply:
Then at least write comment/doc on what's purpose of that type and why it shouldn't be used, in case some curious users will navigate there

`@InternalSerializationApi` does not fit, too?

Reply:
Sure, will drop a comment